core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -2001,6 +2005,115 @@ class WorkspaceService(
       }
     } yield {}
 
+  def getWorkspaceSettings(workspaceName: WorkspaceName): Future[List[WorkspaceSetting]] =
```

Comment:
What do you think about placing these methods in a more focused WorkspaceSettingsService class or some such thing -- this service is already very large.

Reply:
I considered it, but I ultimately didn't as I felt that managing settings belonged in the WorkspaceService as much as anything else. Still, I'm not opposed to it and it would certainly make changing it easier given how much Intellij struggles with the WorkspaceService